bundle/jsky.app.ot/src/main/java/jsky/app/ot/editor/GhostEdIterDarkObs.java

```diff
@@ -25,6 +25,7 @@ public GhostEdIterDarkObs() {
             @Override public void setValue(int newValue) { getDataObject().setStepCount(newValue); }
         });
 
+        System.out.println("*** ADDING WATCHERS ***");
```

Comment:
If you keep this for now, maybe we should flag it with a comment or something that makes it easy to find and remove later.

Reply:
Gah. You're right. This is such a bad habit of mine. I'll flag with TODO and check my TODOs before making a PR.